Completion/X/Command/_kdeconnect-cli

```diff
@@ -0,0 +1,35 @@
+#compdef kdeconnect-cli
+
+_kdeconnect-cli() {
+  _kdeconnect-cli-devices-id() {
+    compadd $(kdeconnect-cli --list-available --id-only)
+  }
+
+  _arguments \
+    '(-l --list-devices)'{-l,--list-devices}'[List all devices]' \
+    '(-a --list-available)'{-a,--list-available}'[List available (paired and reachable) devices]' \
+    '--id-only[Make --list-devices or --list-available print only the devices id, to ease scripting]' \
+    '--refresh[Search for devices in the network and re-establish connections]' \
+    '--pair[Request pairing to a said device]' \
+    '--ring[Find the said device by ringing it.]' \
+    '--unpair[Stop pairing to a said device]' \
+    '--ping[Sends a ping to said device]' \
+    '--ping-msg:Same as ping but you can set the message to display:' \
```

Comment:
This can be constructed more nicely. For example:
`'--ping-msg[send a ping to a device with a custom message to display]:message' \`

Does it allow `--ping-msg=message` or only `--ping-msg message`?